protocol/protobuf/community_token_metadata.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+option go_package = "./;protobuf";
+package protobuf;
+
+message CommunityTokenMetadata {
+  map<uint64, string> contract_addresses = 1;
+  string description = 2;
+  string image = 3;
+}
```

Comment:
I decided to keep it separately because it is bigger now and just in case we want to use it as a separated messages one day. Though, I am not sure these reasons are convincing :)